### PR TITLE
fix(TCAS/ND): TCAS targets disappearing

### LINKED ISSUE #8219: TCAS targets disappearing under different map ranges
### Aircraft Version

Development

### Build info

```json
{
    "built": "2023-09-16T06:20:39+00:00",
    "ref": "refs/heads/master",
    "sha": "4783ebcadace8462bc71af4f8c1bb4c1ebdb322c",
    "actor": "2hwk",
    "event_name": "manual",
    "pretty_release_name": "master:4783ebca",
    "version": "v0.11.0-dev.4783ebc"
}
```

### Describe the bug

While flying in busy airspace, I noticed that the ND was rather empty. When I reduced the range, some TCAS targets appeared that were not there. When I increased map range they disappeared again.

### Expected behavior

The top 8 targets should appear in the ND regardless of selected range. They should not disappear with higher ranges.

### Steps to reproduce

1. Fly in busy airspace
2. Increase/decrease the ND range and notice the TCAS targets

### Additional info (optional)

This was noted in VATSIM

### Discord Username (optional)

Bernardo R

### PR DESCRIPTION
**Pull request by ArtemisPlayer**
[BIS] I accidentally closed the [previous PR](https://github.com/flybywiresim/aircraft/pull/8457), so I made a new one.

Fixes #8219

## Summary of Changes
The mask for displaying TCAS targets on the ND was misplaced (offset of 227 along the Y axis) and that made some targets to disappear/not being displayed. This especially happened when changing ND range.

## Screenshots (if necessary)

Plot of ARC mask (using blue diamonds) on the ND:
image

Full mask on a python plot:
image

ROSE mask on a python plot:
image

Corrected mask:
image

I aligned the ARC mask on the ROSE one.

## Additional context

Discord username : ArtemisPlayer

## Testing instructions
Fly in a busy airspace, such that there is a plane at roughly the same altitude right ahead (about 10 NM), set the ND to 20 NM and ARC mode, and then increase the range: the target should no longer disappear.

## How to download the PR for QA

Every new commit to this PR will cause a new A32NX artifact to be created, built, and uploaded.

1. Make sure you are signed in to GitHub
1. Click on the **Checks** tab on the PR
1. On the left side, click on the bottom **PR** tab
1. Click on the **A32NX** download link at the bottom of the page

### CODE REVIEW

**Comment by lukecologne** [comment]
You can just reopen your old PR?

**Comment by ArtemisPlayer** [comment]
> You can just reopen your old PR?

I tried, but as I deleted the corresponding files, I don't know how to do. If you look at the old PR it says that 0 commits are to be merged.

**Comment by lukecologne** [comment]
Maybe it got confused because you did your changes on master, and now pushed new changes to it. For the future: PRs/changes should only ever be done on new branches

**Comment by 2hwk** [comment]
@tracernz for review (again?)

**Review by 2hwk — APPROVED** [review]

**Review by tracernz — APPROVED** [review]

File: .github/CHANGELOG.md [hunk 1]
```diff
@@ -64,6 +64,7 @@
 1. [AP] Increased nz limit for SRS mode during go-arounds - @aguther (Andreas Guther)
 1. [AP] Further tuned Autoland in Crosswind Situations - @aguther (Andreas Guther)
 1. [EFB] Automatically unfocus EFB input fields when the camera shifts away from the EFB - @tracernz (Mike)
+1. [ND/TCAS] Fixed disappearing TCAS targets - @ArtemisPlayer (ArtemisPlayer)
```
Comment (tracernz) [inline .github/CHANGELOG.md:67]: The changelog line should be added to the end of the release (and kept there when the conflict is solved each time master is merged).

**Comment by 2hwk** [comment]
QA Report

Testing:

- [x] Fly in a busy airspace, such that there is a plane at roughly the same altitude right ahead (about 10 NM), set the ND to 20 NM and ARC mode, and then increase the range: the target should no longer disappear.

Negatives:
- N/A

Results:
- Passed

Conclusions:
- LGTM

**Pull request merged**